polyfills.js

```diff
@@ -19,6 +19,9 @@ process.chdir = function(d) {
   cwd = null
   chdir.call(process, d)
 }
+if (chdir.disabled) {
+ process.chdir.disabled = chdir.disabled
+}
```

Comment:
As a general rule I think we would be better to follow the approach used for `fs.read`.
```suggestion
process.chdir.__proto__ = chdir
```

This way any known/unknown/future properties attached to the original `process.chdir` end up exposed.

Reply:
Thanks, makes sense